nixos/modules/misc/ids.nix

```diff
@@ -187,6 +187,7 @@ in
       #seeks = 148; # removed 2020-06-21
       prosody = 149;
       i2pd = 150;
+      systemd-coredump = 151;
```

Comment:
What was uid 151 before?

Reply:
I don't know. It's probably gone for a while…

As seen above, I doubt it really matters.